src/textual/css/_styles_builder.py

```diff
@@ -1003,6 +1005,30 @@ def process_grid_size(self, name: str, tokens: list[Token]) -> None:
         else:
             self.error(name, tokens[0], "expected two integers here")
 
+    def process_overlay(self, name: str, tokens: list[Token]) -> None:
```

Comment:
Missing a docstring.

Reply:
None of the process methods do. Will leave it until we add all of them.